docs-2.0/k8s-operator/4.cluster-administration/4.7.security/4.7.1.enable-mtls.md

```diff
@@ -493,34 +502,92 @@ Details about these fields are as follows:
 
-!!! caution
-    Ensure that the keys for the certificates and private keys in the created Secret match the values of the `clientCert`, `clientKey`, and `caCert` fields in `spec.sslCerts`. If you've created a Secret of the TLS type, the default key names are `tls.crt` and `tls.key` for the certificate and private key. In this case, you'll need to manually configure `caCert` field with `ca.crt`.
+- In the [chart configuration file of NebulaGraph Operator](../../3.operator-management/3.1.customize-installation.md), [update](../../3.operator-management/3.2.update-operator.md) the `controllerManager.env` value to set the certificate environment variables.
 
-The `insecureSkipVerify` field is used to determine whether the client will verify the server's certificate chain and hostname. The default value is `false`. In production environments, it is recommended to set this to `false` to ensure the security of communication. If set to `true`, the client will not verify the server's certificate chain and hostname.
+  The Nebula-Operator chart allows configuration of the init container, sidecar container, volumes, and volumeMounts within the controller-manager Pod. It accesses the certificates generated by the sidecar-container through the environment variables `CA_CERT_PATH`, `CLIENT_CERT_PATH`, and `CLIENT_KEY_PATH`. To generate certificates, you need to place the sidecar-container and the controller-manager container in the same Pod and share the certificate directory credentials. The controller-manager then reads the client-side certificates from the environment variables and establishes a mutual TLS (mTLS) connection with the Meta and Storage services. Example:
+
+  ```yaml
+  ...
+  controllerManager:
+    env:
+    - name: CA_CERT_PATH
+      value: /credentials/root.crt  # Must use absolute paths.
+    - name: CLIENT_CERT_PATH
+      value: /credentials/client.crt # Must use absolute paths.
+    - name: CLIENT_KEY_PATH
+      value: /credentials/client.key # Must use absolute paths.
+
+    extraInitContainers:
+    - name: init-auth-sidecar
+      command:
+      - /bin/sh
+      - -c
+      args:
+      - cp -R /certs/* /credentials/
+      imagePullPolicy: Always
+      image: reg.vesoft-inc.com/cloud-dev/nebula-certs:latest
+      volumeMounts:
+      - name: credentials
+        mountPath: /credentials
+
+    sidecarContainers:
+      auth-sidecar:
+        imagePullPolicy: Always
+        image: reg.vesoft-inc.com/cloud-dev/nebula-certs:latest
+        volumeMounts:
+          - name: credentials
+            mountPath: /credentials
+
+    extraVolumes:
+      - name: credentials
+        emptyDir:
+          medium: Memory
+
+    extraVolumeMounts:
+      - name: credentials
+        mountPath: /credentials
+  ```
+  
+- In the cluster configuration file, create a Secret to pass in the certificates. Example:
```

Comment:
This is another options. You may add `Alternatively` in this sentence.

Reply:
Should the following configurations be an alternative for Operator?

Reply:
Yes, it's "either..or" between using secrets or environmental variable.

Reply:
Please make it clear that it's `either...or` from line 513 to line 557.

Reply:
Synced offline. Can resolve this.